src/ir/comp.rs

```diff
@@ -532,7 +578,7 @@ impl CompInfo {
         cursor.visit(|cur| {
             if cur.kind() != CXCursor_FieldDecl {
                 if let Some((ty, _)) = maybe_anonymous_struct_field {
-                    let field = Field::new(None, ty, None, None, None, false);
+                    let field = Field::new(None, ty, None, None, None, false, None);
```

Comment:
This is not right, we should be able to know the field offset and track it in `maybe_anonymous_struct_field`.

Does the offset function return something when either called on the struct declaration (where we create the `maybe_anonymous_struct_field`?